src/app/components/how-we-work/how-we-work.component.scss

```
@@ -0,0 +1,312 @@
+@import '../../../styles/variables.scss';
+
+.work {
```

сделай вложенность классов

```suggestion
.work {
	margin-bottom: 134px;
	margin: 0 auto;
	max-width: $desktop;
	@media (max-width: $desktop) {
		padding: 0 15px;
	}
         &__bubble {
	    z-index: -1;
	    position: absolute;
	    width: 315px;
	    height: 348px;
	    background-image: url('../../../assets/svg/bubble.svg');
	    background-repeat: no-repeat;
	    background-size: contain;
	    right: 36%;
	    @media (max-width: $desktop) {
		display: none;
	    }
      }
}
```